src/pybind/mgr/cephadm/services/monitoring.py

```diff
@@ -403,6 +404,7 @@ class PrometheusService(CephadmService):
     DEFAULT_MGR_PROMETHEUS_PORT = 9283
     USER_CFG_KEY = 'prometheus/web_user'
     PASS_CFG_KEY = 'prometheus/web_password'
+    PROMETHEUS_CERT_CFG_KEY = 'prometheus/cert'
```

Comment:
there's no such certificate in the last code.. prom certificates are generated on the fly and not stored anywhere